app/oauth.go

```diff
@@ -257,24 +257,27 @@ func (a *App) GetOAuthAccessTokenForCodeFlow(clientId, grantType, redirectUri, c
 			return nil, model.NewAppError("GetOAuthAccessToken", "api.oauth.get_access_token.redirect_uri.app_error", nil, "", http.StatusBadRequest)
 		}
 
-		if result := <-a.Srv.Store.User().Get(authData.UserId); result.Err != nil {
+		result = <-a.Srv.Store.User().Get(authData.UserId)
+		if result.Err != nil {
 			return nil, model.NewAppError("GetOAuthAccessToken", "api.oauth.get_access_token.internal_user.app_error", nil, "", http.StatusNotFound)
-		} else {
-			user = result.Data.(*model.User)
 		}
+		user = result.Data.(*model.User)
 
-		if result := <-a.Srv.Store.OAuth().GetPreviousAccessData(user.Id, clientId); result.Err != nil {
+		result = <-a.Srv.Store.OAuth().GetPreviousAccessData(user.Id, clientId)
+		if result.Err != nil {
 			return nil, model.NewAppError("GetOAuthAccessToken", "api.oauth.get_access_token.internal.app_error", nil, "", http.StatusInternalServerError)
-		} else if result.Data != nil {
+		}
+
+		if result.Data != nil {
 			accessData := result.Data.(*model.AccessData)
 			if accessData.IsExpired() {
-				if access, err := a.newSessionUpdateToken(oauthApp.Name, accessData, user); err != nil {
+				access, err := a.newSessionUpdateToken(oauthApp.Name, accessData, user)
+				if err != nil {
 					return nil, err
-				} else {
-					accessRsp = access
 				}
+				accessRsp = access
 			} else {
-				//return the same token and no need to create a new session
+				// Return the same token and no need to create a new session
```

Comment:
What's the reason for changing capitalizing the non-godoc comments? Is this another best practice?

Reply:
This is the way the std lib does it. I would consider it best practice. See https://golang.org/doc/effective_go.html#commentary

Reply:
Actually there are a lot of [examples in the standard library of comments that do not begin with capitals](https://github.com/golang/go/blob/master/src/math/exp.go#L146) and my guess would be because capitals have the added export meaning. I don't see a problem with these changes but I would hardly call it best practice when it's a non-documenting comment. And of course if it is documenting an un-exported comment we wouldn't want it to be capitalized.

Reply:
Interesting. Your correct, calling it "best practice" is wrong. It seams to be a personal style choice [as some file use capitalization](https://github.com/golang/go/blob/master/src/net/conf.go).

> And of course if it is documenting an un-exported comment we wouldn't want it to be capitalized.

I don't get that you are saying. Can you please rephrase?

Reply:
Edited my typo: ~~comment~~ code

Reply:
I meant to write un-exported code (not comment). However, upon further reflection Godoc doesn't really care if you start comments for un-exported code with any specific keyword since it doesn't include it in the docs anyhow, so that point is moot. For example it doesn't matter if the comment for this method starts with `userFromLdapEntry` or `UserFromLdapEntry` or `This method...` or `this method...`.

```go
// hello userFromLdapEntry foo bar
func (m *LdapInterfaceImpl) userFromLdapEntry(ldapEntry *ldap.Entry) *model.User {
    ...
```